CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Optimizely Java X SDK Changelog
 
+## [4.0.1-beta]
+August 28th, 2023
+
+### New Features 
+- Update Github Issue Templates ([#531](https://github.com/optimizely/java-sdk/pull/531)).
```

Comment:
This chore change may not be an interest to general audiences. We can remove it or other sections at the bottom.

Reply:
I think we should keep it in CHANGELOG, but under different title like "Functionality Enhancements" below "Bugs Fixed".